Source/WebKit/Platform/cocoa/XPCUtilities.mm

```diff
@@ -39,4 +41,9 @@ void terminateWithReason(xpc_connection_t connection, ReasonCode, const char*)
 ALLOW_DEPRECATED_DECLARATIONS_END
 }
 
+void terminateWithReason(pid_t pid, ReasonCode reasonCode, const char* reason)
+{
+    terminate_with_reason(pid, OS_REASON_WEBKIT, static_cast<uint64_t>(reasonCode), reason, OS_REASON_FLAG_NO_CRASH_REPORT);
```

Comment:
Are you certain this really works? I see to remember Alex trying this in the past and it not working because the system wouldn't let us kill other processes by PID, even child processes.

Reply:
Ah, good point, I think the blocker has been resolved, looking closer.

Thanks for reviewing!

Reply:
I hope you validated locally that this is actually working on both macOS and iOS.

Reply:
I have only validated this on iOS, since we are still using XPC services on macOS. In the XPC case, this behavior cannot be enabled, I believe.

Thanks for reviewing!